lua/super-commit/window/open.lua

```diff
@@ -1,30 +1,33 @@
+local git_cmds = require('super-commit/git/commands')
+local map_table = git_cmds.map_table
+
 local M = {}
 
 function M.single_vertical()
   vim.cmd('vsplit')
   vim.cmd('enew')
 end
 
-M.bufnum_table ={}
 
 function M.setup()
-  local bufnum_1 = vim.api.nvim_get_current_buf();
   vim.cmd('rightbelow vsplit')
   vim.cmd('wincmd l')
   vim.cmd('enew')
-  local bufnum_2 = vim.api.nvim_get_current_buf();
+  map_table["git-diff-init"].bufnum =  vim.api.nvim_get_current_buf();
+  map_table["git-diff-init"].winnum =  vim.api.nvim_win_get_number(0);
```

Comment:
Define this routine process outside.
And as I commented in `super-commit.lua`, holding initial command keys in a set improves readability.

```suggestion
function M.set_current_num(key)
  map_table[key].bufnum = vim.api.nvim_get_current_buf();
  map_table[key].winnum = vim.api.nvim_win_get_number(0);
 end
```

Reply:
Fixed it in https://github.com/supersaintex/super-commit.nvim/commit/3189900d61664541177854465639614cf10efd0a

Reply:
Fixed it in 3189900d61664541177854465639614cf10efd0a